tests/rt_35d.conf

```diff
@@ -0,0 +1,9 @@
+COMPILE | CCPP=Y SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled S2S=Y  |   | orion.intel | fv3 | |
```

Comment:
Can we add hera.intel here and in the rt_wave_35d.conf?

Reply:
Will do. How about without wave?

Reply:
The wave input is also present in the BM_IC on hera. These are just links to the actual BM input except for the 20130401 date.